libs/blocks/quiz/quiz.js

```diff
@@ -332,6 +341,14 @@ export default async function init(
 ) {
   const configData = initConfigPathGlob(el);
   const updatedShortQuiz = shortQuiz || configData.shortQuiz;
+  let storedQuizState = localStorage.getItem('stored-quiz-state') || {};
```

Comment:
I don't think you need the default, as you'd set it in the catch block if JSON.parse would fail